src/operator/loss_binary_op.cc

```diff
@@ -9,7 +9,25 @@ namespace mxnet {
 namespace op {
 
 NNVM_REGISTER_OP(softmax_cross_entropy)
-.MXNET_DESCRIBE("Calculate cross_entropy(data, one_hot(label))")
+.describe(R"code(Calculate cross entropy loss
+between the softmax output of input arrays and one_hot encoding of labels.
+
+.. math::
+   cross_entropy(\mathbf{z}) = \sum_{c=1}^C log(softmax((\mathbf{z})_c) * target_c
+
+This function will convert labels to one_hot encoding first, so
+
+.. math::
+   texttt{cross\_entropy}(\mathbf{z}) = log(softmax(\mathbf{z})_c)
+
+Example::
+
+  >>> data = mx.nd.array([[1, 2, 3], [11, 7, 5]])
```

Comment:
Please don't have python specific constructs in the example here. The examples in the C++ documentation must be language neutral, they will show up in other language bindings as well.